pkg/cluster/node.go

```diff
@@ -353,8 +354,15 @@ func NodeWaitForLogMessage(ctx context.Context, runtime runtimes.Runtime, node *
 		if nRead > 0 && strings.Contains(output, message) {
 			break
 		}
+
+		// check if the container is restarting
+		running, status, _ := runtime.GetNodeStatus(ctx, node)
+		if running && status == k3d.ContainerRestartingStatus {
+			return errors.New(node.Name + " container is restarting, early exit to avoid crash loop")
```

Comment:
```suggestion
			return fmt.Errorf("Node %s is restarting, early exit to avoid crash loop", node.Name)
```
.. for consistency :+1: